data-service/build.gradle

```diff
@@ -38,6 +53,37 @@ compileJava {
     ]
 }
 
+sourceSets {
+    integTest {
+        java.srcDir file('src/integTest/java')
```

Comment:
Would we have integration tests that are not Cucumber features? If not, I don't think another `integTest` SourceSet is actually necessary, since the Cucumber plugin [already sets up the `cucumberTest` SourceSet for us.](https://github.com/samueltbrown/gradle-cucumber-plugin/blob/master/src/main/groovy/com/excella/gradle/cucumber/CucumberPlugin.groovy#L31).

If we do need a separate `integTest` SourceSet (i.e. if we want to run integration tests that need a separate dependency closure from both unit tests and cucumber tests), then it looks like the [`gradle-testsets` plugin](https://github.com/unbroken-dome/gradle-testsets-plugin) would turn all this boilerplate into a single config stanza.

Reply:
This was just moved up from below, it's not new and will be removed once the integ tests are moved into cucumber.